client/src/controllers/ActionController.stories.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import { StimulusWrapper } from '../../storybook/StimulusWrapper';
+import { ActionController } from './ActionController';
+
+export default {
+  title: 'Shared / ActionController',
+  argTypes: {
+    debug: {
+      control: 'boolean',
+      defaultValue: false,
+    },
+  },
+};
+
+const definitions = [
+  {
+    identifier: 'w-action',
+    controllerConstructor: ActionController,
+  },
+];
+
+const Template = ({ debug = false }) => {
+  return (
+    <StimulusWrapper debug={debug} definitions={definitions}>
+      <button
```

Comment:
You may need some kind of listener to handle and block the POST submission. This way we can show that it works without having to trigger a page reload.

Reply:
It doesn't reloads, it redirects to a new url😪😪
But @thibaudcolas said he'll take a look at it

Reply:
OK - let's pull this out to a separate task on the project board tonight, may need to be cleaned up in a separate PR. I have a few ideas so we may be able to do this together on our call.